### PR TITLE
Better welcome message

### LINKED ISSUE #97: Better welcome message when student enters classroom
When a student enters the classroom, they see a nearly blank page with text that just welcomes them. We should add a message that says the next step is for the teacher to pair them.  

Example: 
image

### PR DESCRIPTION
**Pull request by karlrez**
Closes #97 

Added a box to include the welcome message when a student joins a classroom:

image

When student gets removed:
image

### CODE REVIEW

**Comment by karlrez** [comment]
Changed the colors in the last commit (same background color as the chatbox):

image

File: src/components/pages/StudentsPage/WelcomeMessage.css.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
+  background: #f8e5e0;
+  color: red;
+  width: 98%;
+  max-width: 800px;
+  border: 1px solid white;
+  border-radius: 10px;
+  padding: 70px 0;
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.css.tsx:10]: I think the code is cleaner to read when just using padding top and bottom.  Setting the horizontal padding to 0 can confuse whoever is reading the code as padding defaults to 0.
```suggestion
  padding-top: 70px;
  padding-bottom: 70px;
```

File: src/components/pages/StudentsPage/WelcomeMessage.css.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.css.tsx:3]: We should be consistent with our naming. Let's use the same variable name as the filename. 
```suggestion
const welcomeMessageContainer = css`
```

File: src/components/pages/StudentsPage/WelcomeMessage.css.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  color: red;
+  width: 98%;
+  max-width: 800px;
+  border: 1px solid white;
+  border-radius: 10px;
+  padding: 70px 0;
+  text-align: center;
+`;
+
+const unpairedMessageCSS = {
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.css.tsx:14]: For consistency, lets use the same variable name as the fiename.
```suggestion
const welcomeMessageCSS = {
```

File: src/components/pages/StudentsPage/WelcomeMessage.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource @emotion/react */
+
+import { Box, Typography, Link } from '@mui/material';
+
+import unpairedMessageCSS from './WelcomeMessage.css';
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.tsx:5]: Renaming this variable name as per previous comments
```suggestion
import welcomeMessageCSS from './WelcomeMessage.css';
```

File: src/components/pages/StudentsPage/WelcomeMessage.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+export default function WelcomeMessage({ classroom, removed }) {
+  return (
+    <Box css={unpairedMessageCSS.unpairedMessageContainer}>
+      {removed ? (
+        <>
+          <Typography variant='h4' sx={{ mb: 4 }}>
+            Your teacher has removed you from the classroom.
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.tsx:13]:
```suggestion
            Your teacher removed you from the classroom.
```

File: src/components/pages/StudentsPage/WelcomeMessage.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+          </Typography>
+          <Typography variant='h4'>
+            Please return to the Frempco <Link href='/'>homepage</Link>
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.tsx:16]:
```suggestion
            Please return to the <Link href='/'>homepage</Link>
```

File: src/components/pages/StudentsPage/WelcomeMessage.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource @emotion/react */
+
+import { Box, Typography, Link } from '@mui/material';
+
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.tsx:4]: Lets use the `Link` from our shared components folder
```suggestion
import { Box, Typography } from '@mui/material';

import Link from '@components/shared/Link';
```

File: src/components/pages/StudentsPage/index.tsx [hunk 8]
```diff
@@ -65,15 +66,18 @@ export default function StudentsPage({ classroomName }: ClassroomProps) {
...
-        <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        {chatInSession ? (
           <Chatbox socket={socket} chat={chat} setChat={setChat} />
-        </Box>
-      )}
+        ) : (
+          <WelcomeMessage
+            classroom={classroomName}
+            removed={removedFromClass}
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/index.tsx:77]: I'd prefer we keep the original variable names. They feel more descriptive to me. 
```suggestion
            classroomName={classroomName}
            removedFromClass={removedFromClass}
```

File: src/components/pages/StudentsPage/WelcomeMessage.css.tsx [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
+  background: #f8e5e0;
+  color: red;
+  width: 98%;
+  max-width: 800px;
+  border: 1px solid white;
```
Comment (mssiegel) [inline src/components/pages/StudentsPage/WelcomeMessage.css.tsx:8]: This border  is so small I didn't notice it. We can delete it. 
```suggestion
```

**Review by mssiegel — CHANGES_REQUESTED** [review]
I absolutely love the color choices!! Seeing the chatbox lookalike sends a shiver through my body. My adrenaline soars as I start imagining the chat I'm about to enter. Using the chatbox colors and shape is BRILLIANT and brings the student experience to a WHOLE NEW LEVEL!!!

This code is clean and I like that the `WelcomeMessage` its own component.  I left some comments about minor things like renaming variables.

**Comment by karlrez** [comment]
Wow appreciate the comments!! Updated as per your suggestions

**Review by mssiegel — APPROVED** [review]
Looks great! Good to merge!

**Pull request merged**